python/lsst/analysis/drp/histPlot.py

```diff
@@ -0,0 +1,406 @@
+                                               dimensions=("tract", "skymap"))
+
+
+class HistPlotTaskConfig(pipeBase.PipelineTaskConfig,
+                         pipelineConnections=HistPlotTaskConnections):
+
+    leftPanelActions = ConfigurableActionStructField(
+        doc="The actions used to calculate the values for each histogram in the left panel.",
+        default={"hist1": SingleColumnAction,
+                 "hist2": SingleColumnAction},
+    )
+
+    rightPanelActions = ConfigurableActionStructField(
+        doc="The actions used to calculate the values for each histogram in the right panel.",
+        default={"hist1": calcFunctors.SNCalculator,
+                 "hist2": calcFunctors.SNCalculator},
+    )
+
+    leftPanelLabels = pexConfig.DictField(
+        doc="The labels associated with each histogram in the left panel. These dict keys should match "
+        "those in leftPanelActions. If no match is found, a placeholder label will be assigned instead.",
+        keytype=str,
+        itemtype=str,
+        default={"hist1": "PSFlux", "hist2": "9ApFlux"},
+    )
+
+    rightPanelLabels = pexConfig.DictField(
+        doc="The labels associated with each histogram in the right panel. These dict keys should match "
+        "those in rightPanelActions. If no match is found, a placeholder label will be assigned instead.",
+        keytype=str,
+        itemtype=str,
+        default={"hist1": "PSFlux SN", "hist2": "9ApFlux SN"},
+    )
+
+    axisLabels = pexConfig.DictField(
+        doc="Axis labels for both x-axes, and one for the single unified y-axis.",
+        keytype=str,
+        itemtype=str,
+        default={"xLeft": "flux (nJy)", "xRight": "S/N", "yLeft": "frequency"},
+    )
+
+    summaryStatsLabel = pexConfig.Field(
+        doc="Name of the column used to generate the on-sky summary plot. Should be one of the keys in "
+        "leftPanelLabels/rightPanelLabels. If no match is found, set to the first key in leftPanelLabels.",
+        dtype=str,
+        default="9ApFlux",
+    )
+
+    selectorActions = ConfigurableActionStructField(
+        doc="Which selectors to use to narrow down the data for QA plotting.",
+        default={"flagSelector": dataSelectors.FlagSelector},
+    )
+
+    pLower = pexConfig.Field(
+        doc="Percentile used to determine the lower range of the histogram bins. If more than one histogram "
+        "is plotted per-panel, the percentile limit is the minimum value across all input data.",
+        dtype=float,
+        default=2.0,
+    )
+
+    pUpper = pexConfig.Field(
+        doc="Percentile used to determine the upper range of the histogram bins. If more than one histogram "
+        "is plotted per-panel, the percentile limit is the maximum value across all input data.",
+        dtype=float,
+        default=98.0,
+    )
+
+    nBins = pexConfig.Field(
+        doc="Number of bins used to divide the x axes.",
+        default=40,
+        dtype=int,
+    )
```

Comment:
As I commented on colorColorFitPlot.py, it is useful to be able to explicitly set the bin width rather than the number of bins. It's easier to read and compare histograms when you know the width of the bins.